modules/backend/console/CreateController.php

```diff
@@ -69,11 +68,16 @@ protected function prepareVars(): array
         $vars['model'] = $model;
         $vars['sidebar'] = $this->option('sidebar');
 
-        $layout = $this->option('sidebar') ? 'sidebar' : 'standard';
+        if ($this->option('stubs')) {
+            $this->stubs['scaffold/controller/index.stub'] = 'controllers/{{lower_name}}/index.php';
+            $this->stubs['scaffold/controller/_list_toolbar.stub'] = 'controllers/{{lower_name}}/_list_toolbar.php';
 
-        $this->stubs["scaffold/controller/{$layout}/create.stub"] = 'controllers/{{lower_name}}/create.php';
-        $this->stubs["scaffold/controller/{$layout}/update.stub"] = 'controllers/{{lower_name}}/update.php';
-        $this->stubs["scaffold/controller/{$layout}/preview.stub"] = 'controllers/{{lower_name}}/preview.php';
+            $layout = $this->option('sidebar') ? 'sidebar' : 'standard';
```

Comment:
We'll need to add support for this as a property on the controller class ($formLayout perhaps?) and then have the default views look at that property before rendering and have the property set in the scaffold. 

Options would be "standard" (can be not set), "sidebar" (the my account page with secondaryTabs rendered on the right), or "fancy" (the CMS / static pages layout).